pmacApp/pmc/trajectory_scan_code_ppmac.pmc

```diff
@@ -194,7 +189,31 @@ Return
 // *************************************************************************************************
 
 N103:
-    CalculatedBase = CurrentBufferAdr + CurrentIndex
+    CalculatedBase = CurrentBufferAdr/4
```

Comment:
(note for myself)
Substitute for 
`CalculatedBase = CurrentBufferAdr/_SIZEOF_INTEGER_`